src/foundation.cs

```diff
@@ -8532,11 +8532,19 @@ interface NSObject2 : NSObjectProtocol {
 		// NSPlaceholders (informal) protocol
 		[Static]
 		[Export ("defaultPlaceholderForMarker:withBinding:")]
+#if XAMCORE_4_0 && MONOMAC
+		NSObject GetDefaultPlaceholder (NSObject marker, NSBindingSelectionMarker binding);
+#else
 		NSObject GetDefaultPlaceholder (NSObject marker, NSString binding);
+#endif
 
 		[Static]
 		[Export ("setDefaultPlaceholder:forMarker:withBinding:")]
+#if XAMCORE_4_0 && MONOMAC
+		void SetDefaultPlaceholder (NSObject placeholder, NSBindingSelectionMarker marker, NSString binding);
+#else
 		void SetDefaultPlaceholder (NSObject placeholder, NSObject marker, NSString binding);
+#endif
```

Comment:
it's weird that `marker` became `NSBindingSelectionMarker` here while it was `binding` in the previous one !?! logically I'd say the first one should be

```
NSObject GetDefaultPlaceholder (NSBindingSelectionMarker marker, NSString binding);
```

Reply:
So here's the diff in question:

```-+ (void)setDefaultPlaceholder:(nullable id)placeholder forMarker:(nullable id)marker withBinding:(NSBindingName)binding;    // marker can be nil or one of NSMultipleValuesMarker, NSNoSelectionMarker, NSNotApplicableMarker
-+ (nullable id)defaultPlaceholderForMarker:(nullable id)marker withBinding:(NSBindingName)binding;    // marker can be nil or one of NSMultipleValuesMarker, NSNoSelectionMarker, NSNotApplicableMarker
++ (void)setDefaultPlaceholder:(nullable id)placeholder forMarker:(nullable NSBindingSelectionMarker *)marker withBinding:(NSBindingName)binding;    // marker can be nil or one of multipleValuesSelectionMarker, noSelectionMarker, notApplicableSelectionMarker
++ (nullable id)defaultPlaceholderForMarker:(nullable NSBindingSelectionMarker *)marker withBinding:(NSBindingName)binding;    // marker can be nil or one of multipleValuesSelectionMarker, noSelectionMarker, notApplicableSelectionMarker
```

Reply:
In other words, I made a mistake.

Reply:
so
```
+ (nullable id)defaultPlaceholderForMarker:(nullable NSBindingSelectionMarker *)marker withBinding:(NSBindingName)binding;    // marker can be nil or one of multipleValuesSelectionMarker, noSelectionMarker, notApplicableSelectionMarker
```
is not
```
NSObject GetDefaultPlaceholder (NSObject marker, NSBindingSelectionMarker binding);
```
but the binding I guessed, right ?

Reply:
I believe it is fixed now.